libraries/botbuilder-runtime-integration-express/src/index.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as t from 'runtypes';
+import express from 'express';
+import { Configuration, getRuntimeServices } from 'botbuilder-runtime';
+import { IServices, ServiceCollection } from 'botbuilder-runtime-core';
+
+/**
+ * Options for runtime Express adapter
+ */
+const TypedOptions = t.Record({
+    /**
+     * Port that server should listen on
+     */
+    port: t.Number,
+
+    /**
+     * Path that the server will listen to for [Activities](xref:botframework-schema.Activity)
+     */
+    messagingEndpointPath: t.String,
+});
+
+export type Options = t.Static<typeof TypedOptions>;
```

Comment:
Is the top-level comment (`Options for runtime Express adapter`) "exposed" for `type Options`?

Reply:
It was not, the top-level comment needs to be above the `export type Options = ...`, fixed!